src/epcpm/tests/test_project.py

```diff
@@ -71,7 +74,7 @@ def test_model_values():
     for name in model:
         model[name] = name + "_"
 
-    assert tuple(model.values()) == ("parameters_", "can_", "sunspec_")
+    assert tuple(model.values()) == ("parameters_", "can_", "staticmodbus_", "sunspec_")
```

Comment:
doesnt matters much but I think the order should be consistent with the interface tab order

Reply:
I reordered references to static modbus in the code in [this commit](https://github.com/altendky/pm/pull/10/commits/7e2fc74e272b327be111df54ccf02fdead1d5481).